test/integration/render-tests/runtime-styling/layout-property-default-to-property-expression/style.json

```diff
@@ -0,0 +1,78 @@
+{
+  "version": 8,
+  "metadata": {
+    "test": {
+      "width": 64,
+      "height": 64,
+      "operations": [
+        [
+          "setLayoutProperty",
+          "symbol",
+          "icon-rotate",
+          [
```

Comment:
Can we improve the conversion so this is just `["get", "rotate"]`?

Reply:
👍  sneaking that in here, as well as exporting function-related modules in the main style-spec package (i'd done this locally to get a build of the style spec that i could use in the node conversion script)